src/Controller/ApiV1Controller.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Tatoeba Project, free collaborative creation of multilingual corpuses project.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * @category PHP
+ * @package  Tatoeba
+ * @license  Affero General Public License
+ * @link     https://tatoeba.org
+ */
+namespace App\Controller;
+
+use App\Controller\AppController;
+use App\Form\SentencesSearchForm;
+use App\Model\CurrentUser;
+use Cake\Core\Configure;
+use Exception;
+
+/**
+ * Controller for Api.
+ *
+ * @category Sentences
+ * @package  Controllers
+ * @author   githubshrek
+ * @license  Affero General Public License
+ * @link     https://tatoeba.org
+ */
+class ApiV1Controller extends AppController
+{
+    /**
+     * Show Json of specified sentence id.
+     * 
+     * @param mixed $id Id of the sentence.
+     * 
+     * @return void
+     */
+    public function sentence($id = null) {
+        // We will handle the output directly, so as not to display the header and footer
+        $this->disableAutoRender();
+
+        $this->loadModel('Sentences');
+
+        if (is_numeric($id)) {
+            // Retrieve the sentence
+            $sentence = $this->Sentences->getSentenceWith($id, [
+                'translations' => true,
+                'sentenceDetails' => true
+            ]);
+
+
+            return $this->getResponse()
+                ->withStringBody($sentence);
```

Comment:
I'd suggest to return this instead:
```
    return $this->response
            ->withType('application/json')
            ->withStringBody($sentence);
```

And with this you can normally remove `$this->disableAutoRender();` at the beginning.

Reply:
OK, looks great. I notice when making this change that the result seems to display in a user-friendly indented way.

![image](https://user-images.githubusercontent.com/7943077/120241223-922df100-c262-11eb-9626-3d7667d11bf3.png)